x/posts/internal/keeper/post_response.go

```diff
@@ -10,15 +10,15 @@ import (
 // that is returned to user upon a query
 type PostQueryResponse struct {
 	types.Post
-	Likes    types.Likes   `json:"likes"`
-	Children types.PostIDs `json:"children"`
+	Reactions types.Reactions `json:"reactions"`
+	Children  types.PostIDs   `json:"children"`
 }
 
-func NewPostResponse(post types.Post, likes types.Likes, children types.PostIDs) PostQueryResponse {
+func NewPostResponse(post types.Post, reactions types.Reactions, children types.PostIDs) PostQueryResponse {
```

Comment:
I guess the query from CLI is not getting this post response as it's still querying the post only.

https://github.com/desmos-labs/desmos/blob/4f43ef8ae3dfea83de9bbeeba3c8cd3fbc43683b/x/posts/client/cli/query.go#L39

Reply:
Fixed this